textproc/scdoc/files/patch-makefile.diff

```diff
@@ -0,0 +1,12 @@
+--- Makefile.orig	2019-03-04 16:43:31.000000000 +0100
++++ Makefile	2019-06-04 12:56:32.000000000 +0200
+@@ -1,8 +1,7 @@
+ VERSION=1.9.4
+ CFLAGS+=-g -DVERSION='"$(VERSION)"' -Wall -Wextra -Werror -Wno-unused-parameter
+-LDFLAGS+=-static
+ INCLUDE+=-Iinclude
+-PREFIX?=/usr/local
++PREFIX?=/opt/local
```

Comment:
The MacPorts prefix is not guaranteed to be /opt/local, so this should not be hardcoded. See https://trac.macports.org/wiki/PortfileRecipes#hardcode_opt_local

Reply:
Thanks very much. I figured as much but wasn't able to find this information in the documentation. In this regard the documentation could be improved.

So from what I gather, `reinplace` is a TCL command that does a simple regex-replace?

Reply:
Done.